RELEASE_NOTES

```diff
@@ -3,6 +3,54 @@ For more detailed information, please see the git log.
 
 These release notes can also be consulted at http://easybuild.readthedocs.org/en/latest/Release_notes.html.
 
+v3.0.0 (November 16th 2016)
+---------------------------
+
+feature release
+- backward-incompatible changes:
+  - make robot always consider subtoolchains, even without --minimal-toolchains (but in reverse order) (#1973)
+  - clean up behaviour that was deprecated for EasyBuild v3.0 (#1978)
+  - change default config to use Lmod/Lua for modules tool/syntax, GC3Pie as job backend (#1985)
+  - the minimal required version of Lmod was bumped to 5.8
```

Comment:
see #1985, it's mostly about code cleanup actually... I'll just include a ref to the PR in which the requirement was bumped.